2020/prep/submitted/make_spaces.sh

```diff
@@ -111,20 +115,28 @@ done
 
 # Single Query Track
 [[ -n $OUT_SPACE_SQ ]] && \
-python $PREPARE "$IN_SPACE_NON_INC" "$SOLVERS_CSV" "$OUT_SPACE_SQ" -t single_query --select "$SELECT_SQ" -w
+python $PREPARE "$IN_SPACE_NON_INC" "$SOLVERS_CSV" "$OUT_SPACE_SQ" \
+    -t single_query --select "$SELECT_SQ" -w \
+    $(echo $INCLUDE_NONCOMPETITIVE)
```

Comment:
The `echo` is unnecessary:
```suggestion
    $INCLUDE_NONCOMPETITIVE
```

Just be sure to not include quotes, so that no argument is added when the variable is emty.